### PR TITLE
Add new ui test to quay 

### PR DESCRIPTION
**Pull request by huweihua-redhat**
What does this PR do:

* Add quay new UI test

### CODE REVIEW

**Comment by huweihua-redhat** [comment]
/pj-rehearse  periodic-ci-quay-quay-tests-master-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse  periodic-ci-quay-quay-tests-new_ui_add_tag-quay-newui-quay-e2e-tests-quay311-ocp416-newui

File: ci-operator/step-registry/quay-tests/test-quay-newui/quay-tests-test-quay-newui-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+ref:
+  as: quay-tests-test-quay-newui
+  cli: latest
+  from_image:
```
Comment (kuiwang02) [inline ci-operator/step-registry/quay-tests/test-quay-newui/quay-tests-test-quay-newui-ref.yaml:4]:
```yaml
from: quay-test-console
```

File: ci-operator/config/quay/quay-tests/quay-quay-tests-new_ui_add_tag__quay-newui.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
```
Comment (kuiwang02) [inline ci-operator/config/quay/quay-tests/quay-quay-tests-new_ui_add_tag__quay-newui.yaml:11]:
```yaml
images:
- dockerfile_literal: |
    FROM src
    RUN curl --silent --location  https://rpm.nodesource.com/setup_lts.x | bash - && \
        curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
        PACKAGES="openssh-clients httpd-tools nodejs yarn xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver alsa-lib skopeo" && \
        yum install --setopt=tsflags=nodocs -y $PACKAGES && \
        yum install -y yum-utils && yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo && \
        yum -y install terraform \
        yum clean all && rm -rf /var/cache/yum/*
    RUN curl -Lo firefox-latest.tar.bz2 'https://download.mozilla.org/?product=firefox-latest&os=linux64&lang=en-US' && \
        tar xjf firefox-*.tar.bz2 && \
        mv firefox /opt && \
        ln -s /opt/firefox/firefox /usr/local/bin/firefox && \
        firefox -v
  from: src
  to: quay-test-console
```

File: ci-operator/config/quay/quay-tests/quay-quay-tests-new_ui_add_tag__quay-newui.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
+build_root:
```
Comment (kuiwang02) [inline ci-operator/config/quay/quay-tests/quay-quay-tests-new_ui_add_tag__quay-newui.yaml:6]: @huweihua-redhat 
I checked the quay-test-master file and it is 
```yaml
build_root:
  image_stream_tag:
    name: release
    namespace: openshift
    tag: golang-1.21
```

now the docker building is failing, maybe related to this.

**Comment by kuiwang02** [comment]
@huweihua-redhat I checked failure. and find it is introduced by https://github.com/openshift/release/pull/52867.
it change `https://rpm.nodesource.com/setup_16.x` to `https://rpm.nodesource.com/setup_lts.x` and then it does not succeed any more.

@LiZhang19817 
when you test #52867, you should try `/pj-rehearse  pull-ci-quay-quay-tests-master-images` while you try `/pj-rehearse periodic-ci-quay-quay-tests-master-ocp-416-quay-quay-e2e-tests-quay311-ocp416-lp-interop` which does not build image and just import image, so you do not meet the failure.
because you change images part, need to execute `/pj-rehearse  pull-ci-quay-quay-tests-master-images` to see if it is ok.


@huweihua-redhat so, you could change it back to `https://rpm.nodesource.com/setup_16.x` in this PR to try to test others.

File: ci-operator/config/quay/quay-tests/quay-quay-tests-new_ui_add_tag__quay-newui.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+images:
+- dockerfile_literal: |
+    FROM src
+    RUN curl --silent --location  https://rpm.nodesource.com/setup_lts.x | bash - && \
```
Comment (kuiwang02) [inline ci-operator/config/quay/quay-tests/quay-quay-tests-new_ui_add_tag__quay-newui.yaml:14]: `https://rpm.nodesource.com/setup_lts.x` to `https://rpm.nodesource.com/setup_16.x` to continue to test your PR. 
after it is fixed, you could change it back.

**Comment by huweihua-redhat** [comment]
/pj-rehearse  ci/rehearse/quay/quay-tests/new_ui_add_tag/quay-newui-images

**Comment by huweihua-redhat** [comment]
/pj-rehearse  ci/rehearse/periodic-ci-quay-quay-tests-new_ui_add_tag-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-new_ui_add_tag-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-new_ui_add_tag-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-new_ui_add_tag-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Comment by huweihua-redhat** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-new_ui_add_tag-quay-newui-quay-e2e-tests-quay311-ocp416-newui

**Pull request closed without merging**